Test_Automation_Framework/Epam_TestAutomation_BusinessLogic/PageObjects/Pages/JoinOurTeamPages.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Epam_TestAutomation_Core.BasePage;
+using Epam_TestAutomation_Core.Browser;
+using Epam_TestAutomation_Core.Elements;
+using Epam_TestAutomation_Core.Helper;
+using Epam_TestAutomation_Core.Utils;
+using Epam_TestAutomation_TestData.JoinOurTeamInfo;
+using OpenQA.Selenium;
+
+namespace Epam_TestAutomation_BusinessLogic.PageObjects.Pages
+{
+    public class JoinOurTeamPages : BasePage
+    {
+        public Button CareersBlog => new Button(By.XPath("//*[@href='/careers/blog']"));
+
+        public Link CareerButton => new Link(By.XPath("//*[@href = '/careers']"));
+
+        public Button JobListingsButton => new Button(By.XPath("//*[@href = '/careers/job-listings']"));
+
+        public TextInput KeywordInput => new(By.XPath("//*[@id='new_form_job_search_1445745853_copy-keyword']"));
+
+        public Button FindButton => new(By.XPath("//*[@type='submit']"));
+
+        public Dropdown LocationDropdown => new Dropdown(By.XPath("//*[@class='select2-selection select2-selection--single']"));
+
+        public Dropdown DropdownLocations => new Dropdown(By.XPath("//*[@class='select2-results__options open']"));
+
+        public TextInput LocationInput => new TextInput(By.XPath("//*[@class='select2-search__field']"));
+
+        public Button CitiesLineButton => new Button(By.XPath("//*[@class='select2-results__option select2-results__option--highlighted']"));
+
+        public Label SkillsLabel => new Label(By.XPath("//*[@class='default-label']"));
+
+        public Label SkillFilter => new Label(By.XPath("//*[@class='filter-tag']"));
+
+        public Dropdown SkillsDropdown => new Dropdown(By.XPath("//*[@class='multi-select-dropdown' and @aria-hidden='false']"));
```

Comment:
no need to add and @aria-hidden='false'. Element can be found by [@class='multi-select-dropdown']